SampleViewer/mainSample.cpp

```diff
@@ -26,6 +26,10 @@
 #include <QQuickWindow>
 #include <QtGlobal>
 
+#ifdef Q_OS_LINUX
```

Comment:
```suggestion
#if defined(Q_OS_LINUX) && !defined(Q_OS_ANDROID)
```
Same for the ifdef logic below.